src/components/Vaults/index.tsx

```diff
@@ -26,8 +27,8 @@ const Vaults = ({ label, requiredCapacity, isShown, onSelectionCallback, error }
     // Filters out vaults with lower than required capacity and sorts by accountId
     // to have vaults with same accountId grouped together.
     const vaultsWithEnoughCapacity = allVaults
-      .filter((vault) => vault[1].gt(BitcoinAmount.from.Satoshi(0)))
-      .filter((vault) => vault[1].gte(BitcoinAmount.from.Satoshi(requiredCapacity)))
+      .filter((vault) => vault[1].gt(BitcoinAmount.zero()))
```

Comment:
@bvotteler 
Could you please review this change? Does it look correct? `BitcoinAmount.from.Satoshi(0)` -> `BitcoinAmount.zero()`?

Reply:
> Does it look correct? BitcoinAmount.from.Satoshi(0) -> BitcoinAmount.zero()?
That part: LGTM 👍 

I'm just a bit confused why there are 2 filter statements. Are there cases where `requiredCapacity` is less than zero?

Reply:
Thanks for confirming. I'm not sure exactly why since I just ported the original code focusing on the lib changes regarding the currency refactoring.
But your concern makes a lot of sense. The first filter looks redundant. For now, I'm going to leave a TODO comment on it since I do not want to mix that with the main currency refactoring changes. Later on, I think we can revisit it by reading that TODO comment. In addition, it does not look like it causes any critical issue.